feature-tests/tests/resources/features/aisGET/InvokeApiGateWay-HappyPath.feature

```diff
@@ -19,69 +19,79 @@ Feature: Invoke-APIGateway-HappyPath.feature
+            | pswResetRequired      | userActionIdResetSuccess  | true         | AIS_FORCED_USER_PASSWORD_RESET                     | false        | true           | true          | false           | 04          | FRAUD_FORCED_USER_PASSWORD_RESET                             |
+            | pswResetRequired      | userActionPswResetSuccess | true         | AIS_FORCED_USER_PASSWORD_RESET                     | false        | true           | true          | false           | 04          | FRAUD_FORCED_USER_PASSWORD_RESET                             |
+            | pswResetRequired      | unSuspendAction           | true         | AIS_FORCED_USER_PASSWORD_RESET                     | false        | true           | true          | false           | 04          | FRAUD_FORCED_USER_PASSWORD_RESET                             |
             #suspend no action account status to new intervention type
-            | suspendNoAction          | pswResetRequired          | false        | AIS_FORCED_USER_PASSWORD_RESET                     | false        | true           | true          | false           |
-            | suspendNoAction          | suspendNoAction           | false        | AIS_ACCOUNT_SUSPENDED                              | false        | true           | false         | false           |
-            | suspendNoAction          | block                     | false        | AIS_ACCOUNT_BLOCKED                                | true         | false          | false         | false           |
-            | suspendNoAction          | idResetRequired           | false        | AIS_FORCED_USER_IDENTITY_VERIFY                    | false        | true           | false         | true            |
-            | suspendNoAction          | pswAndIdResetRequired     | false        | AIS_FORCED_USER_PASSWORD_RESET_AND_IDENTITY_VERIFY | false        | true           | true          | true            |
-            | suspendNoAction          | unblock                   | false        | AIS_ACCOUNT_SUSPENDED                              | false        | true           | false         | false           |
-            | suspendNoAction          | userActionIdResetSuccess  | false        | AIS_ACCOUNT_SUSPENDED                              | false        | true           | false         | false           |
-            | suspendNoAction          | userActionPswResetSuccess | false        | AIS_ACCOUNT_SUSPENDED                              | false        | true           | false         | false           |
-            | suspendNoAction          | unSuspendAction           | false        | AIS_ACCOUNT_SUSPENDED                              | false        | true           | false         | false           |
```

Comment:
If user is in Suspend No user Action (suspended: true, everything else is false) and an Unsuspend intervention is received (code 02), all flags should return as false